src/bootstrap.tsx

```diff
@@ -75,11 +88,11 @@ const checkUserVerification = async () => {
       e.status === 401 &&
       e.response?.data?.errorCode === 'NDEx_User_Account_Not_Verified'
     ) {
-      const userInfo: UserInfo = (await keycloak.loadUserInfo()) as UserInfo
+      const userInfo = parseMessage(e.response?.data?.message)
```

Comment:
@GAOChengzhan How come we no longer call loadUserInfo here?

Reply:
Oh, Jing wants me to get the info from the error message directly instead of calling this function which is not up-to-date